test/models/user_test.rb

```diff
@@ -26,6 +26,28 @@
     end
   end
 
+  describe "#time_format" do
+  let(:user) { User.create!(name: "jimbob", email: 'test@test.com') }
+    it "should have a default time format of relative" do
+      user.time_format.must_equal('relative')
+    end
+
+    it "should not update" do
+      lambda {user.update_attributes!(:time_format => 'foobar') }.must_raise ActiveRecord::RecordInvalid
+    end
+
+    it "should update" do
+      user.update_attributes!(:time_format => 'utc')
+      user.reload
+      user.time_format.must_equal('utc')
+    end
+
+    local_user = User.create!(name: "bettysue", email: 'bsue@test.com', time_format: 'local') 
```

Comment:
creating things outside of it/describe means they will not get rolled back and will stack in the test database -> move this into to `it` or a `before`